app/javascript/elm/src/Main.elm

```diff
@@ -633,15 +637,17 @@ view model =
         ]
 
 
-viewSearchAsIMove : Bool -> Bool -> Html Msg
-viewSearchAsIMove wasMapMoved isSearchAsIMoveOn =
+viewSearchAsIMove : Bool -> Bool -> Path -> Html Msg
+viewSearchAsIMove wasMapMoved isSearchAsIMoveOn resetIcon =
     div [ class "search-control" ]
         [ if wasMapMoved then
             button
                 [ class "button button--primary search-control__button"
                 , Events.onClick FetchSessions
                 ]
-                [ text "Redo Search in Map" ]
+                [ text "Redo Search in Map"
+                , img [ src (Path.toString resetIcon), alt "Reset icon" ] []
```

Comment:
You can also use `<|` as in `src <| Path.toString resetIcon`

Reply:
https://github.com/HabitatMap/AirCasting/commit/9b64befbe5d86e8261bf6bf0e4c16e4d7dd25a52